docs/DeveloperGuide.md

```diff
@@ -1009,14 +1009,14 @@ Currently, the user can only delete one company at once. However, there will be
 
 **Proposed Enhancement**
 
-Enable the user to input multiple indices when attempting to delete entries. Users can separate each index with a comma.
-The `DeleteCommandParser` will then split the string by commas and remove the companies corresponding to the specified indices. There will also be checks to see if the user keyed in the same index more than once. If the same index is keyed in more than once, the parser will accept the input but treat it as if the user only keyed in that same index once.
+Enable the user to input multiple indices when attempting to delete entries. Users can separate each index with a comma. If the user wishes to delete a range of indices, they can use a dash, to indicate the range. The format to delete a range of companies: `INDEX_START - INDEX_END`.
+The `DeleteCommandParser` will split the string by commas and remove the companies corresponding to the specified indices. There will also be checks to see if the user keyed in the same index more than once. If the same index is keyed in more than once, the parser will accept the input but treat it as if the user only keyed in that same index once.
```

Comment:
add in also, if the INDEX_START or INDEX_END falls outside the acceptable range of indices, it'll also be an error

see if want to specify what's the error, whether it's greater than number of companies or not positive integer... (aka same error messages still apply to each INDEX_START and INDEX_END)